src/build.c

```diff
@@ -1063,7 +1063,7 @@ gboolean build_parse_make_dir(const gchar *string, gchar **prefix)
 		gchar *input;
 
 		/* get the start of the path */
-		pos = strstr(string, "/");
+		pos = strchr(pos, "/");
```

Comment:
```suggestion
		pos = strchr(pos, '/');
```
When using `strchr`, use single quotes around the char (because it's a char and not a string)

Reply:
Thanks! Fixed now. :)